packages/opentelemetry-resources/package.json

```diff
@@ -55,6 +60,7 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^0.5.2",
-    "@opentelemetry/base": "^0.5.2"
+    "@opentelemetry/base": "^0.5.2",
+    "gcp-metadata": "^3.5.0"
```

Comment:
Minor: latest available version is `4.0.0`.

Reply:
It appears as though node 8 support was dropped in `4.0.0`: https://github.com/googleapis/gcp-metadata/blob/master/CHANGELOG.md#-breaking-changes.